DependentExtensions/Swig/MakeSwig.bat

```diff
@@ -6,7 +6,19 @@ if "%swigPath%"=="" goto :SKIPADDSLASH
 if "%swigPath:~-1%"=="\" goto :SKIPADDSLASH 
 SET swigPath=%swigPath%\
 :SKIPADDSLASH
-del /F /Q SwigOutput\SwigCSharpOutput\*
+
+set swigOutputCsharp=SwigOutput\SwigCSharpOutput
+set swigOutputCplusDLL=SwigOutput\CplusDLLIncludes
+set swigInterface=SwigInterfaceFiles
+set swigFiles=SwigWindowsCSharpSample\SwigTestApp\SwigFiles
+
+if not exist %swigOutputCsharp% ( mkdir %swigOutputCsharp% )
+if not exist %swigOutputCplusDLL% ( mkdir %swigOutputCplusDLL% )
+if not exist %swigInterface% ( mkdir %swigInterface% )

```

Comment:
The SwigInterfaceFiles directory should exist prior to running the MakeSwig file since it should contain the required interface files. Hence, creating this directory here should not really serve any purpose or am I overlooking anything?